Flaky tests on macOS

All of `TestGetObject` subtests have been randomly failing for the last couple of days.
- The tests seem to only be failing on macos
- Reverting to 3 months back (fe6663b) still triggers the errors (which had never happened in the past) so it's not due to a recent code change
- The error messages vary
- The version of macOS on the CI hasn't changed (`10.15.7`), but the version of the Virtual Environment did (`20201017.1` => `20201130.3`)
- Can repro on Big Sur


```
--- FAIL: TestGetObject (0.00s)
    --- FAIL: TestGetObject/valid_object_should_return_an_object (0.03s)
        --- FAIL: TestGetObject/valid_object_should_return_an_object/commit (0.01s)
            /app/git-go/ginternals/packfile/packfile_test.go:76: 
                	Error Trace:	packfile_test.go:76
                	Error:      	Received unexpected error:
                	            	could not get base object at offset 18446744073709538867:
                	            	    github.com/Nivl/git-go/ginternals/packfile.(*Pack).getObjectAt
                	            	        /app/git-go/ginternals/packfile/packfile.go:289
                	            	  - could not seek from 0 to object offset 18446744073709538867:
                	            	    github.com/Nivl/git-go/ginternals/packfile.(*Pack).getRawObjectAt
                	            	        /app/git-go/ginternals/packfile/packfile.go:127
                	            	  - seek /var/folders/mh/s6jhkzqd2s18jn4wj0sccbr00000gn/T/TestGetObject_valid_object_should_return_an_object_214944132/.git/objects/pack/pack-0163931160835b1de2f120e1aa7e52206debeb14.pack: invalid argument
                	Test:       	TestGetObject/valid_object_should_return_an_object/commit
        --- FAIL: TestGetObject/valid_object_should_return_an_object/tree (0.00s)
            /app/git-go/ginternals/packfile/packfile_test.go:119: 
                	Error Trace:	packfile_test.go:119
                	Error:      	Received unexpected error:
                	            	could not get zlib reader:
                	            	    github.com/Nivl/git-go/ginternals/packfile.(*Pack).getRawObjectAt
                	            	        /app/git-go/ginternals/packfile/packfile.go:245
                	            	  - zlib: invalid header
                	Test:       	TestGetObject/valid_object_should_return_an_object/tree
        --- FAIL: TestGetObject/valid_object_should_return_an_object/blob (0.00s)
            /app/git-go/ginternals/packfile/packfile_test.go:104: 
                	Error Trace:	packfile_test.go:104
                	Error:      	Received unexpected error:
                	            	unknown object type 0:
                	            	    github.com/Nivl/git-go/ginternals/packfile.(*Pack).getRawObjectAt
                	            	        /app/git-go/ginternals/packfile/packfile.go:167
                	Test:       	TestGetObject/valid_object_should_return_an_object/blob
FAIL
coverage: 59.6% of statements
FAIL	github.com/Nivl/git-go/ginternals/packfile	0.479s
FAIL
```

--- a/ginternals/packfile/packfile_test.go
+++ b/ginternals/packfile/packfile_test.go
@@ -68,8 +68,6 @@ func TestGetObject(t *testing.T) {
 
 		// TODO(melvin): Test multiple parents
 		t.Run("commit", func(t *testing.T) {
-			t.Parallel()
-
 			commitOid, err := ginternals.NewOidFromStr("1dcdadc2a420225783794fbffd51e2e137a69646")
 			require.NoError(t, err)
 			o, err := pack.GetObject(commitOid)
@@ -96,8 +94,6 @@ func TestGetObject(t *testing.T) {
 		})
 
 		t.Run("blob", func(t *testing.T) {
-			t.Parallel()
-
 			blobOid, err := ginternals.NewOidFromStr("3f2f87160d5b4217125264310c22bcdad5b0d8bb")
 			require.NoError(t, err)
 			o, err := pack.GetObject(blobOid)
@@ -111,8 +107,6 @@ func TestGetObject(t *testing.T) {
 		})
 
 		t.Run("tree", func(t *testing.T) {
-			t.Parallel()
-
 			treeOid, err := ginternals.NewOidFromStr("c799e9129faae8d358e4b6de7813d6f970607893")
 			require.NoError(t, err)
 			o, err := pack.GetObject(treeOid)
@@ -136,8 +130,6 @@ func TestGetObject(t *testing.T) {
 		})
 
 		t.Run("tag", func(t *testing.T) {
-			t.Parallel()
-
 			// TODO(melvin): we now support tags
 			t.Skip("tags not yet supported")
 		})

